<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ROISelectionWindow</class>
 <widget class="QDialog" name="ROISelectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>527</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>395</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="ConfirmPB">
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="UndoPB">
      <property name="text">
       <string>Undo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clearAllPB">
      <property name="text">
       <string>Clear All</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="switchPB">
      <property name="text">
       <string>Switch</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
